sql/core/src/test/resources/sql-tests/results/pgSQL/window.sql.out

```diff
@@ -0,0 +1,2381 @@
+-- !query 83 output
+1	3
+1	3
+1	3
+11	11
+13	13
+15	15
+17	17
+19	19
+21	21
+23	23
+25	25
+27	27
+29	29
+31	31
+33	33
+35	35
+37	37
+39	39
+41	41
+43	43
+45	45
+47	47
+5	5
+7	7
+9	9
+
+
+-- !query 84
+WITH cte (x) AS (
+        select 1 union all select 1 union all select 1 union all
+        SELECT * FROM range(5, 49, 2)
+)
+SELECT x, (sum(x) over w)
+FROM cte
+WINDOW w AS (ORDER BY x range between 1 preceding and 1 following)
+-- !query 84 schema
+struct<x:bigint,sum(x) OVER (ORDER BY x ASC NULLS FIRST RANGE BETWEEN CAST((- 1) AS BIGINT) FOLLOWING AND CAST(1 AS BIGINT) FOLLOWING):bigint>
+-- !query 84 output
+1	3
+1	3
+1	3
+11	11
+13	13
+15	15
+17	17
+19	19
+21	21
+23	23
+25	25
+27	27
+29	29
+31	31
+33	33
+35	35
+37	37
+39	39
+41	41
+43	43
+45	45
+47	47
+5	5
+7	7
+9	9
+
+
+-- !query 85
+SELECT count(*) OVER (PARTITION BY four) FROM (SELECT * FROM tenk1 UNION ALL SELECT * FROM tenk2)s LIMIT 0
+-- !query 85 schema
+struct<count(1) OVER (PARTITION BY four ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING):bigint>
+-- !query 85 output
+
+
+
+-- !query 86
+create table t1 (f1 int, f2 int) using parquet
+-- !query 86 schema
+struct<>
+-- !query 86 output
+
+
+
+-- !query 87
+insert into t1 values (1,1),(1,2),(2,2)
+-- !query 87 schema
+struct<>
+-- !query 87 output
+
+
+
+-- !query 88
+select f1, sum(f1) over (partition by f1 order by f1
+                         range between 1 preceding and 1 following)
+from t1 where f1 = f2
+-- !query 88 schema
+struct<f1:int,sum(f1) OVER (PARTITION BY f1 ORDER BY f1 ASC NULLS FIRST RANGE BETWEEN 1 PRECEDING AND 1 FOLLOWING):bigint>
+-- !query 88 output
+1	1
+2	2
+
+
+-- !query 89
+explain
+select f1, sum(f1) over (partition by f1 order by f2
+range between 1 preceding and 1 following)
+from t1 where f1 = f2
+-- !query 89 schema
+struct<plan:string>
+-- !query 89 output
+== Physical Plan ==
+*Project [f1#x, sum(f1) OVER (PARTITION BY f1 ORDER BY f2 ASC NULLS FIRST RANGE BETWEEN 1 PRECEDING AND 1 FOLLOWING)#xL]
++- Window [sum(cast(f1#x as bigint)) windowspecdefinition(f1#x, f2#x ASC NULLS FIRST, specifiedwindowframe(RangeFrame, -1, 1)) AS sum(f1) OVER (PARTITION BY f1 ORDER BY f2 ASC NULLS FIRST RANGE BETWEEN 1 PRECEDING AND 1 FOLLOWING)#xL], [f1#x], [f2#x ASC NULLS FIRST]
+   +- *Sort [f1#x ASC NULLS FIRST, f2#x ASC NULLS FIRST], false, 0
+      +- Exchange hashpartitioning(f1#x, 200), true
+         +- *Project [f1#x, f2#x]
+            +- *Filter ((isnotnull(f1#x) AND isnotnull(f2#x)) AND (f1#x = f2#x))
+               +- *ColumnarToRow
+                  +- FileScan parquet default.t1[f1#x,f2#x] Batched: true, DataFilters: [isnotnull(f1#x), isnotnull(f2#x), (f1#x = f2#x)], Format: Parquet, Location: InMemoryFileIndex[file:/home/dguedes/Workspace/spark/sql/core/spark-warehouse/org.apache.spark.sq..., PartitionFilters: [], PushedFilters: [IsNotNull(f1), IsNotNull(f2)], ReadSchema: struct<f1:int,f2:int>
```

Comment:
Oh. It seems that we should not do this. This has the physical location with your id. This cannot succeed in the Jenkins or someone else's computer.
```
InMemoryFileIndex[file:/home/dguedes/Workspace
```

Please comment this `EXPLAIN` statement too. And, add simple comment about this. For this, I believe we don't need a new JIRA.